src/schemas/response.py

```
@@ -1,10 +1,19 @@
 # src/schemas/response.py
 from pydantic import BaseModel
-from typing import Optional, Any
+from typing import Optional, Any, List
```

Ruff (0.14.8)

`typing.List` is deprecated, use `list` instead

(UP035)